sdk/core/abort-controller/karma.conf.js

```diff
@@ -42,20 +42,22 @@ module.exports = function(config) {
     // test results reporter to use
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ["mocha", "coverage", "remap-coverage", "junit"],
+    reporters: ["mocha", "coverage", "karma-remap-istanbul", "junit"],
```

Comment:
Is `remap-coverage` same as `karma-remap-coverage` ?I thought you wanted to replace the latter with `karma-remap-istanbul`

Reply:
That's the expected configuration by the plugins.

Refer to [karma-remap-coverage#configuration](https://github.com/sshev/karma-remap-coverage/blob/master/README.md#configuration) and [karma-remap-istanbul#configuration](https://github.com/marcules/karma-remap-istanbul#configuration)